### PR TITLE
feat: implemented delete file and delete directory features

### PR DESCRIPTION
**Pull request by IgorShadurin**
Implemented #44

### CODE REVIEW

File: test/integration/fdp-class.spec.ts [hunk 1]
```diff
@@ -243,6 +243,25 @@ describe('Fair Data Protocol class', () => {
...
+
+    it('should delete a directory', async () => {
+      const fdp = createFdp()
+      const user = generateUser(fdp)
+      const pod = generateRandomHexString()
+      const directoryName = generateRandomHexString()
+      const directoryFull = '/' + directoryName
+      await topUpAddress(fdp)
+
+      await fdp.account.register(user.username, user.password)
```
Comment (nugaon) [inline test/integration/fdp-class.spec.ts:255]: why is it necessary to register the user?

**Review by nugaon — APPROVED** [review]
Nice  : )

please amend the documentation about the delete actions before the merge.

**Comment by tfius** [comment]
nice !!

File: test/integration/fdp-class.spec.ts (hunk 1 above)
Comment (IgorShadurin) [reply test/integration/fdp-class.spec.ts:255]: User registration in each individual test is required for independent storage testing

Comment (nugaon) [reply test/integration/fdp-class.spec.ts:255]: but a simple wallet generation wouldn't be enough?

Comment (IgorShadurin) [reply test/integration/fdp-class.spec.ts:255]: Created an issue for that https://github.com/fairDataSociety/fdp-storage/issues/59

**Pull request merged**